cmd/antrea-agent/options.go

```diff
@@ -306,6 +306,23 @@ func (o *Options) validateAntreaIPAMConfig() error {
 	return nil
 }
 
+func (o *Options) validateMulticlusterConfig(encapMode config.TrafficEncapModeType) error {
+	if !o.config.Multicluster.Enable {
+		o.config.Multicluster.EnableStretchedNetworkPolicy = false
+		return nil
+	}
+
+	if !features.DefaultFeatureGate.Enabled(features.Multicluster) {
+		return fmt.Errorf("Multicluster can only be enabled when Multicluster featureGate is enabled")
+	}
+
+	if encapMode != config.TrafficEncapModeEncap {
```

Comment:
Again, check `o.config.Multicluster.EnableStretchedNetworkPolicy && encapMode != config.TrafficEncapModeEncap`, only stretched NP require encapMode. @luolanzone

Reply:
Yes, that's also my thoughts. Maybe you missed my previous reply https://github.com/antrea-io/antrea/pull/3914#discussion_r1043257200. Let's see @luolanzone 's reply

Reply:
@jianjuns @GraysonWu yes, `Multicluster.Enable` is for enabling MC Gateway.

Reply:
Ok.